dotcom-rendering/src/components/FrontSectionTitle.tsx

```diff
@@ -113,17 +159,59 @@ export const FrontSectionTitle = ({ title, collectionBranding }: Props) => {
 				isContainerBranding,
 				isFrontBranding,
 			} = collectionBranding;
+			const isAdvertisingPartnerOrExclusive =
+				logo.label.toLowerCase() === 'advertising partner' ||
+				logo.label.toLowerCase() === 'exclusive advertising partner';
 			if (isFrontBranding || isContainerBranding) {
 				return (
 					<div css={titleStyle}>
 						{title}
-						<>
-							<p css={labelStyles}>{logo.label}</p>
-							<Badge imageSrc={logo.src} href={logo.link} />
-							<a href={aboutThisLink} css={aboutThisLinkStyles}>
+						<div
+							css={
+								isAdvertisingPartnerOrExclusive &&
+								inAdvertisingPartnerABTest &&
+								advertisingPartnerDottedBorder
+							}
+						></div>
```

Comment:
I wonder if it might be simpler to conditionally render this element, and whether a horizontal rule `hr` might be more appropriate.

Reply:
I was just following the design from PROD and add on it the new updates that is why I wanted to have the same dotted ruler. I will make it conditionally render for now and ask about the `hr` because everything needs to be approved from a design perspective.

Reply:
Addressed in https://github.com/guardian/dotcom-rendering/pull/11217#discussion_r1607967178